src/services/users.js

```
@@ -0,0 +1,52 @@
+'use strict';
+
+let users = [];
+
+const resetUsers = () => {
+  users = [];
+};
+
+const getAll = () => {
+  return users;
+};
+
+const getById = (userId) => {
+  const foundUser = users.find(user => user.id === userId);
+
+  return foundUser || null;
+};
```

let's add some error handling and throw an error if user not found